src/definitions/modules/shape.js

```diff
@@ -197,6 +198,20 @@ $.fn.shape = function(parameters) {
           },
           hidden: function() {
             return $module.closest(':hidden').length > 0;
+          },
+          mobile: function() {
+            var
+                userAgent    = navigator.userAgent,
```

Comment:
You may want to add a check to see if `navigator` exists, some browsers may not have support https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator#Browser_compatibility

Reply:
I copied the code from sidebar.js. Would need to change this aswell the too. But MA, be I skip the whole check and just use 150ms all the time?

Reply:
Yeah could do, that would be the easiest option.

Reply:
Did it . See latest commit